src/elasticsearch.js

```diff
@@ -0,0 +1,204 @@
+require('array.prototype.flatmap').shim();
```

Comment:
This can be a good reference to see how to rewrite this in module format: https://github.com/aicore/Core-Analytics-Server/blob/main/src/compression-manager.js